index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Microverse Trial Survey Form</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fitno">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css" media="screen" />
+</head>
+
+<body>
+    <main>
+        <div class="header">
```

Comment:
- [ ] _OPTIONAL_ HTML5 semantic is good practice

Reply:
_[OPTIONAL]_

- [ ] I would recommend you use HTML5 `header` tag and other semantic tags in other places.

Semantic HTML uses elements for their given purpose, making it easier for both people and machines to read and understand it. Making applications accessible not only ensures equal access for people with disabilities but also benefits people without disabilities by allowing them to customize their experiences. See this [awesome article](http://diveintohtml5.it/semantics.html) that Microverse recommends to learn more on HTML5.